src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1450,6 +1457,11 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     return isUnsafeQuote ? setShouldShowWarningAcknowledgement(true) : handleSubmit()
   }, [handleSubmit, isUnsafeQuote])
 
+  const handleOnWalletNotConnectedClick = useCallback(() => {
+    walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+    handleBackClick()
```

Comment:
q: looks like this will always route back to pools page, is that intended? i.e:
- Position page: route back to pools
- Standalone "Add Liquidity": also route back to pools

which doesn't sound like the behavior we want in any of the two cases?

https://jam.dev/c/5d3f745b-1ebe-43d5-9445-6fb05614bf5f

Reply:
Removed this behavior

Reply:
Fixed in [1b98335](https://github.com/shapeshift/web/pull/7270/commits/1b98335766e29210cd5969426bc7506fc19efea8)